CHANGES.md

```diff
@@ -16,8 +16,13 @@ development source code and as such may not be routinely kept up to date.
 ## Improvements
 
 * We now produce standalone installation archives for macOS running on aarch64
-  hardware (aka arm64, Apple Silicon, M1/M2).
-  ([#357](https://github.com/nextstrain/cli/pull/357))
+  hardware (aka arm64, Apple Silicon, M1/M2).  The standalone installer will
+  use these archives starting with this release.  Rosetta 2 is still required
+  to be enabled at install time, as it's still needed by Nextstrain's Conda and
+  Docker runtimes.  This requirement may be relaxed in the future since not all
```

Comment:
Nod. It's a TODO for now because if we remove it from the installer, we should add it to the runtime checks. And I didn't want to block on that for now.

Re: Docker, our arm64 image isn't totally free of x86_64 code, at least yet. So Docker needs to emulate for that, and Rosetta can be used to run that x86_64 code faster ([c.f.](https://docs.docker.com/desktop/settings/mac/)).

Reply:
> and Rosetta can be used to run that x86_64 code faster

Absolutely right. But it's not required.

Reply:
Perhaps the thing to do is move the Rosetta check into a) the Conda runtime checks as a requirement and b) the Docker runtime checks as an advisory check (similar to the existing memory check)?

Reply:
Yeah, sounds ok. When installing docker on (arm64) mac it asks you to install rosetta and you have to explicitly tell it not to, so if anyone's in this situation the lack of rosetta is a conscious choice.

Reply:
Ok, see https://github.com/nextstrain/cli/pull/361 for the Rosetta checks at the runtime level instead + the new commit pushed here.